storage/client/client.go

```diff
@@ -272,18 +273,21 @@ func (c *StorageClient) Transactions(ctx context.Context, p apiTypes.GetConsensu
 			&t.Fee,
 			&t.Method,
 			&t.Body,
-			&t.Code,
+			&code,
 			&module,
 			&message,
 			&t.Timestamp,
 		); err != nil {
 			return nil, wrapError(err)
 		}
-		if *t.Code == oasisErrors.CodeNoError {
+		if code == oasisErrors.CodeNoError {
 			t.Success = true
 		} else {
-			t.Module = &module
-			t.Message = &message
+			t.Error = &apiTypes.TxError{
+				Code:    int(code),
```

Comment:
`code` is int32 in the DB, so you should be able to just directly scan into one and not convert here.

Reply:
Actually, how about we fudge it a lil bit relative to the DB and go with `uint32` for consistency with [oasis-sdk](https://github.com/oasisprotocol/oasis-sdk/blob/fb741678585c04fdb413441f2bfba18aafbf98f3/client-sdk/go/types/transaction.go#L490)?